### DISCUSSION TITLE
How to install?

### DISCUSSION THREAD

**Discussion by senryusama** [THUDM/CogVideo#534] [Q&A] [answered]
As per title, how do I install this locally?
I tried downloading CogVideo-1.0.zip, 
the proceeded with the pip install -r requirements.txt on the cmd window, 
only to get and error 
![bandicam 2024-11-20 15-31-22-783](https://github.com/user-attachments/assets/6f3d455b-b7e3-4c08-8b9a-86b787f7fcb3)
The problem is...I already did instal pytorch, using the url mentioned on the screen, but the error won't go away...
What am I doing wrong?

Nevermind, got it to work with pinokio.

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
Windows is unable to perform a normal pip install of deepspeed; you need to find the correct version of deepspeed that is compatible with Windows.
